Doc/source/designspaceLib/readme.rst

```diff
@@ -547,7 +542,7 @@ Attributes
    to the root path of this document. The path can be at the same level
    as the document or lower.
 -  ``layer``: optional, string. The name of the layer in the source file.
-   If no layer attribute is given assume it is the foreground layer.
+   If no layer attribute is given assume the foreground layer should be used.
```

Comment:
actually, the default layer is not necessarily the "foreground" one, see @typesupply 's reply:
https://github.com/unified-font-object/ufo-spec/issues/38#issuecomment-388103944

so if no layer attribute is given, we should use whatever is the current default layer in the UFO (i.e. the one pointing to the default "glyphs" directory). The use of "foreground" here may be confusing.